example/src/screens/TCPServer/reducer.ts

```diff
@@ -59,6 +59,7 @@ export const TCPServerReducer: Reducer = (
             return {
                 ...state,
                 state: ServerState.Ready,
+                port: `${action.payload.port}`,
```

Comment:
If you really need it as string somewhere I would cast it differently and most likely not in reducer...  🤔